README.md

```diff
@@ -129,6 +129,41 @@ Below is a rough sketch of potential interfaces.
   The environment record does not contain a property for any names that are
   imported and reexported without a lexical binding.
 
+A loader privately retains references to:
+
+* A global object
+* A global environment record
+* A static module record promise memo
+* A static module record memo
+* A module instance promise memo
+* A module instance memo
+* A resolve hook
+* A load hook
+* An import meta hook
+
+Every realm retains in itself an intrinsic loader, with a host-defined global
+environment, resolve hook, load hook, and import meta hook.
+Every constructed loader (guest) by default shares the global object, global
+environment record, resolve hook, load hook, and *static* memos of its host.
+Loader constructor options may override these defaults.
+
+Loaders accept a `globals` option.
+If provided, the guest loader will create its own empty global object with a
+null prototype and derrive its own global environment record from that object.
+The loader will then create an intrinsic `eval`, `Function`, and `Loader` that
+refer back to the global environment record, such that scripts and modules
+evaluated in the loader share this global environment record and such that
+direct `eval` in the loader can succeed.
+The loader then copies the own properties of the `globals` option
+over the new `globalThis` using assignment.
```

Comment:
```suggestion
onto the new `globalThis` using assignment.
```

Also, what is meant by "using assignment"? That they become data properties?

Reply:
I’ll clarify that this is `[[Set]]`.